domino_data/data_sources.py

```diff
@@ -640,7 +640,7 @@ def get_datasource(self, name: str) -> Datasource:
 
         error_response = cast(ErrorResponse, response.parsed)
         message = (
-            f"Received unexpected response {response.status_code} while getting data source"
+            f"Received unexpected response {response.status_code} while getting data source: ${response}"
```

Comment:
Print the raw content of the response. Still keep the status_code in the error message cause that's what we know for sure from the response object. In case the raw content is printing something not informational(e.g. print as an object as a particular address), we can still see the status code.